integrations/huggingface-transformers/tutorials/token-classification/token-classification-cli.md

```diff
@@ -0,0 +1,273 @@
+```
+
+### **Export to ONNX**
+
+Once you have trained your model, export to ONNX in order to deploy with DeepSparse. The artifacts of the training process are saved to your local filesystem. 
+
+Run the following to convert your PyTorch checkpoint to ONNX:
+
+```bash
+sparseml.transformers.export_onnx \
+  --model_path ./sparse_bert-token_classification_conll2003 \
+  --task text_classification
+```
+
+A `deployment` folder is created in your local directory, which has all of the files needed for deployment with DeepSparse including the `model.onnx`, `config.json`, and `tokenizer.json` files.
+
+## Sparse Transfer Learning with a Custom Dataset (WNUT_17)
+
+Beyond the Conll2003 dataset, we can also use a dataset from the Hugging Face Hub or pass via local files. Let's try an example of each for the sentiment analysis using [WNUT 17](wnut_17), which is also a NER task.
+
+For simplicity, we will perform the fine-tuning without distillation. Although the transfer learning recipe contains distillation
+modifiers, by setting `--distill_teacher disable` we instruct SparseML to skip distillation.
+
+### WNUT_17 Inspection
+
+Run the following to inspect the Rotten Tomatoes dataset.
+
+```python
+from datasets import load_dataset
+
+wnut_17 = load_dataset("wnut_17")
+print(wnut_17)
+print(wnut_17["train"][0])
+```
+
+We can see that each row dataset contains a `tokens` field which contains a list of strings representing each word the sentence and a corresponding `ner_tags` which is a list of integers representing the tag of each word in the sentence.
+
+### Using a Hugging Face Dataset Identifier
+
+To use this dataset with the CLI, we can replace the `--dataset_name conll2003` argument with `--dataset_name wnut_17 --input_column_names tokens --label_column_name ner_tags`. SparseML will then download the dataset from the Hugging Face hub and run training as before.
+
+```
+sparseml.transformers.token_classification \
+  --model_name_or_path zoo:nlp/masked_language_modeling/obert-base/pytorch/huggingface/wikipedia_bookcorpus/pruned90-none \
+  --recipe zoo:nlp/token_classification/obert-base/pytorch/huggingface/conll2003/pruned90_quant-none \
+  --recipe_args '{"num_epochs":12,"qat_start_epoch":7.0, "observer_epoch": 11.0}' \
+  --distill_teacher disable \
+  --dataset_name wnut_17 --text_column_name tokens --label_column_name ner_tags \
+  --output_dir sparse_bert-token_classification_wnut_17 \
+  --per_device_train_batch_size 32 --per_device_eval_batch_size 32 --preprocessing_num_workers 6 \
+  --do_train --do_eval --evaluation_strategy epoch --fp16 --seed 29204  \
+  --save_strategy epoch --save_total_limit 1
+```
+
+You will notice that we used the same recipe as we did in the Conll2003 case (identified by the SparseZoo stub `zoo:nlp/token_classification/obert-base/pytorch/huggingface/conll2003/pruned90_quant-none`). Since the WNUT and Conll2003 tasks are similiar, we chose to start with the same hyperparameters as we used in Conll2003 training.
```

Comment:
Since the WNUT and Conll2003 tasks are similar, we started with the same hyperparameters we used in Conll2003 training.